src/Components/Application/Product/F&B/AddCustomizationGroup.jsx

```diff
@@ -0,0 +1,208 @@
+
+  const validate = () => {
+    const formErrors = {};
+    formErrors.name =
+      newCustomizationGroupData?.name == undefined || newCustomizationGroupData?.name?.trim() === ""
+        ? "Name is not allowed to be empty"
+        : "";
+    formErrors.minQuantity =
+      newCustomizationGroupData?.minQuantity == undefined || newCustomizationGroupData?.minQuantity === ""
+        ? "Min Quantity is not allowed to be empty"
+        : newCustomizationGroupData?.minQuantity <= 0
+        ? `Please enter a valid quantity`
+        : "";
+    formErrors.maxQuantity =
+      newCustomizationGroupData?.maxQuantity == undefined || newCustomizationGroupData?.maxQuantity === ""
+        ? "Max Quantity is not allowed to be empty"
+        : newCustomizationGroupData?.maxQuantity <= 0
+        ? `Please enter a valid quantity`
+        : "";
+
+    setErrors(formErrors);
+
+    return !Object.values(formErrors).some((val) => val !== "");
+  };
+
+  const handleSubmit = () => {
+    if (validate() === true) {
+      setErrors({});
+      handleAddCustomizationGroup(newCustomizationGroupData, inputType);
+    }
+  };
+
+  const handleChange = (e) => {
+    const selectedValue = e.target.value;
+    setInputType(selectedValue);
+  };
+
+  useEffect(() => {
+    if (props.mode === "edit" && Object.keys(newCustomizationGroupData).length > 0) {
+      setInputType(newCustomizationGroupData.inputType);
+    } else {
+      setInputType("select");
+    }
+  }, [newCustomizationGroupData]);
+
+  return (
+    <div>
+      <Modal
+        open={showModal}
+        onClose={() => {
+          setErrors({});
+          handleCloseModal();
+          setNewCustomizationGroupData({});
+        }}
+      >
+        <div
+          style={{
+            position: "absolute",
+            top: "50%",
+            left: "50%",
+            transform: "translate(-50%, -50%)",
+            backgroundColor: "#fff",
+            padding: "24px 40px",
+            borderRadius: 20,
+          }}
+        >
+          <p className="font-semibold text-xl" style={{ marginBottom: 10 }}>
+            {props.mode === "edit" ? "Edit Customization Group" : "Add New Customization Group"}
+          </p>
+          <div className="flex items-center">
+            <label className="w-40 my-4 text-sm py-2 ml-1 font-medium text-left text-[#606161] inline-block">
+              Name:
+            </label>
+
+            <CssTextField
+              required
+              type={"input"}
+              className="w-80 h-full px-2.5 py-3.5 text-[#606161] bg-transparent !border-black"
+              size="small"
+              autoComplete="off"
+              placeholder={"Enter Customisation Group Name"}
+              error={!!errors.name}
+              helperText={errors.name}
+              value={newCustomizationGroupData.name}
+              onChange={(e) => setNewCustomizationGroupData({ ...newCustomizationGroupData, name: e.target.value })}
+            />
+          </div>
+
```

Comment:
this is repetition of code. Add one function which will take values to be input and return the  CssTextField component 
you can do this change later also. but do it.